### ISSUE TITLE
Getdropdownview displays nothing when MaterialSpinnerBaseAdapter is extended

### ISSUE THREAD

**Issue by suku-h** [jaredrummler/MaterialSpinner#20]
I have tried to use the following code:

```
public class EnglishSpinnerAdapter extends MaterialSpinnerBaseAdapter {
private List list;
private Context ctx;

public EnglishSpinnerAdapter(Context context, List items) {
    super(context);
    ctx = context;
    list = items;
}

@Override
public Object getItem(int position) {
    return list.get(position);
}

@Override
public int getCount() {
    return list.size();
}

@Override
public Object get(int position) {
    return list.get(position);
}

@Override
public List getItems() {
    return list;
}

@Override
public View getView(int position, View convertView, ViewGroup parent) {
    TextView textView = new TextView(ctx);
    textView.setTypeface(S.engFont);
    textView.setTextSize(18);
    textView.setGravity(Gravity.CENTER);
    textView.setTextColor(ContextCompat.getColor(ctx, R.color.black));
    textView.setText(list.get(position).toString());
    return textView;
}

@Override
public View getDropDownView(int position, View convertView, ViewGroup parent) {
    TextView textView = new TextView(ctx);
    textView.setTypeface(S.engFont);
    textView.setTextSize(18);
    textView.setGravity(Gravity.CENTER);
    textView.setTextColor(ContextCompat.getColor(ctx, R.color.black));
    textView.setText(list.get(position).toString());
    return textView;
}
```

}`

I get this output. Get drop down view is blank. But if I select an item, it gets displayed properly. So if I selected the place where February should have been displayed, I get February as the output. So the issue is in just the rendering of the view.

![spinnergetdropdownview](https://cloud.githubusercontent.com/assets/17564687/17646734/1d030376-61f4-11e6-9f0c-0493d8782596.PNG)

**Comment by gooddayashok** [comment]
the same problem in my drop list

**Comment by LiaoFuYun** [comment]
I have the same problem. Did you find a solution?

**Comment by suku-h** [comment]
I had to use the other adapter...MaterialSpinnerAdapter I think that is
it's name

**Comment by LiaoFuYun** [comment]
I solved it, thank you very much

**Comment by mabdallah450** [comment]
@LiaoFuYun I have the same problem. How did you solve it?

**Comment by LiaoFuYun** [comment]
just call setTextColor,like this:
`yourAdapter.setTextColor(Color.parseColor("#333333"));`

**Comment by MichelPro** [comment]
@LiaoFuYun  Sorry, I use ths way that you answer, but no change.

**Comment by jaredrummler** [comment]
@LiaoFuYun is right. Fixed here: a062680b0c9aac61af52fe29796b13cf435f0c82
